nss-day-cohort-22/Class-Website
# fixed typo in my bio, buisiness -> business ; grammar, got -> received

## Link to Ticket

TITLE [Link]() #3 

## Description of Proposed Changes

- Updated Cohort.Json with spelling correction

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout <fix_typo>
start http-server
navigate to my bio
verify that typo is corrected
grammar correction is there. 

```


## Impacted Areas in Application

List general components of the application that this PR will affect:

* cohort.json

## Mentions @username
@john-dulaney @krnorris65 
Tag users that need to review this code
